### ISSUE TITLE
Different names of installation files for desktop versions and mobile versions

### ISSUE THREAD

**Issue by rakleed** [AdguardTeam/AdGuardVPNForAndroid#338]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN version

v2.5 (5)

### Operating mode

VPN

### Environment

```markdown
- OS: Android 14
- Device: Google Pixel 3A
- Firmware: N/A
```

### Root access

- [ ] Yes, I have it.

### Issue Details

Steps to reproduce:
1. Download APK file from https://adguard-vpn.com/en/android/overview.html

### Expected Behavior

The name of the APK file will be spelled correctly as in the installer for macOS: `AdGuardVPN`

### Screenshots

<details><summary>Screenshot 1:</summary>

![Screenshot 2023-09-04 at 05 11 56](https://github.com/AdguardTeam/AdGuardVPNForAndroid/assets/19418601/0db5ea67-5d67-4cca-a982-429ff93c67dc)

</details>

**Comment by maxikuzmin** [comment]
@rakleed yes, you're right, the names of the files are different, at the moment we have a single build name for desktop versions, and a different name for mobile versions, this is not a bug. The term “Installer" for the .apk file is not particularly applicable, so the names are different.
Maybe in the future we'll make a uniform file name format, thanks for your observation!

**Comment by rakleed** [comment]
@maxikuzmin can you leave the issue open and close it only when you make changes?

**Comment by maxikuzmin** [comment]
@rakleed since this is not a bug, and we currently have no need for a common file name standard across OS, I've closed the issue. We can change the status of your issue to "Feature request" for other users to vote on it. Let's see how many likes it will gain.
